allennlp/tests/semparse/domain_languages/wikitables_language_test.py

```diff
@@ -0,0 +1,696 @@
+# pylint: disable=no-self-use,invalid-name,too-many-public-methods
+from typing import List
+
+import pytest
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.data.tokenizers import Token
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.semparse.contexts import TableQuestionContext
+from allennlp.semparse.domain_languages.domain_language import ExecutionError
+from allennlp.semparse.domain_languages.wikitables_language import Date, WikiTablesLanguage
+from allennlp.tests.semparse.domain_languages.domain_language_test import check_productions_match
+
+
+class TestWikiTablesLanguage(AllenNlpTestCase):
```

Comment:
I think it is a good idea to include tests that use the `ActionSpaceWalker` here as well, as we discussed yesterday. It's fine to add them in a different PR, but may be add a TODO?